src/transforms/push.ts

```diff
@@ -64,12 +65,13 @@ export const createJobData = (payload, jiraHost: string): PushQueueMessagePayloa
 		jiraHost,
 		installationId: payload.installation.id,
 		webhookId: payload.webhookId || "none",
-		webhookReceived: payload.webhookReceived || undefined
+		webhookReceived: payload.webhookReceived || undefined,
+		...(!gitHubAppConfig ? undefined : { gitHubAppConfig })
 	};
 };
 
-export const enqueuePush = async (payload: unknown, jiraHost: string) =>
-	await sqsQueues.push.sendMessage(createJobData(payload, jiraHost));
+export const enqueuePush = async (payload: GitHubPushData, jiraHost: string, gitHubAppConfig: GitHubAppConfig | undefined) =>
```

Comment:
Can we use same signature as above for `gitHubAppConfig?: GitHubAppConfig` instead of `... | undefined`

Reply:
fixed.